django_cloud_deploy/tests/cleanup/resource_cleanup.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Versconsolen 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITconsoleNS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissconsolens and
+# limitatconsolens under the License.
+"""Automatic deletion of resources more than 2 hours old on GCP."""
+
+import datetime
+import re
+
+from django_cloud_deploy.tests.lib import test_base
+from django_cloud_deploy.tests.lib import utils
+from googleapiclient import discovery
+
+
+class GCPResourceCleanUp(test_base.ResourceCleanUp, test_base.ResourceList):
+    """Clean up GCP resources more than 2 hours old."""
+
+    MAX_DIFF = datetime.timedelta(hours=2)
+
+    def _should_delete(self, resource_name: str) -> bool:
+        """Return whether the given resource should be deleted."""
+
+        # The resource created for testing should be in format similar with
```

Comment:
This comment should be part of the `args` section of the doc string.

Reply:
Done.